conversion/binary_to_octal.py

```diff
@@ -0,0 +1,29 @@
+def binary_to_octal(msg: str) -> str:
+    """
+    implementasi biner ke oktal
+    >>> binary_to_octal("1111")
+    '17'
+    """
+    if not all(char in "01" for char in msg):
```

Comment:
```suggestion
    if not all(char in "01" for char in binary):
```
ganti menggunakan ``binary``